pkg/hns/endpoint_windows.go

```diff
@@ -17,11 +17,13 @@
 package hns
 
 import (
-	"github.com/Microsoft/hcsshim"
-	"github.com/containernetworking/cni/pkg/types/current"
+	"fmt"
 	"log"
 	"net"
 	"strings"
+
+	"github.com/Microsoft/hcsshim"
```

Comment:
Did you run gofmt?

Reply:
@rakelkar 

I changed these codes in the vscode editor. It looks like that my editor auto run the  gofmt command for me.
should  I manually make it recover as before?